pybamm/discretisations/discretisation.py

```diff
@@ -152,6 +158,43 @@ def process_rhs_and_algebraic(self, model):
         model.algebraic = self.process_dict(model.algebraic)
         model.concatenated_algebraic = self.concatenate(*model.algebraic.values())
 
+    def process_mass_matrix(self, model):
```

Comment:
I think "create_mass_matrix" is a more appropriate name

Reply:
I agree